compiler/z/codegen/OMRLinkage.hpp

```diff
@@ -101,13 +101,13 @@ enum TR_S390LinkageConventions
 #define SkipGPRsForFloatParms         0x040
 #define PadFloatParms                 0x080
 #define TwoStackSlotsForLongAndDouble 0x100
-#define FloatParmDescriptors          0x200
+// Available                          0x200
 #define AggregatesPassedOnParmStack   0x400
 #define AggregatesPassedInParmRegs    0x800
 #define AggregatesReturnedInRegs      0x1000
 // Available                          0x2000
 #define SmallIntParmsAlignedRight     0x4000  ///< < gprSize int parms aligned into the parmword
-#define ParmBlockRegister             0x8000  ///< Has a parameter block register: OS Linkage (non-Java)
```

Comment:
This is only for z/OS OS linkage which doesn't exist in OMR, so deprecating it. We only support XPLINK, and currently have no plans to support other system linkages.